src/clj/rems/middleware.clj

```diff
@@ -202,19 +202,20 @@
      (users/format-user identity))))
 
 (defn wrap-cache-control
-  "In case a Cache-Control header is missing, add a default of 23h"
+  "In case a Cache-Control header is missing, add a default of no-store"
```

Comment:
The name of this could perhaps indicate what it does i.e. `wrap-cache-no-store-if-no-cache-specified`, `wrap-cache-no-store-when-not-specified` or `wrap-cache-default-no-store`.